source/Cosmos.HAL2/BlockDevice/EBR.cs

```diff
@@ -30,7 +30,7 @@ public EBR(byte[] aEBR)
             ParsePartition(aEBR, 462);
         }
 
-        protected void ParsePartition(byte[] aEBR, uint aLoc)
```

Comment:
is this right? just asking as its the only change like this from unsigned to signed and stayed 32 bit